service/controller/key/key.go

```diff
@@ -474,16 +446,24 @@ func NetworkDNSBlock(servers []net.IP) string {
 	return dnsBlock
 }
 
-func NetworkNTPBlock(servers []net.IP) string {
-	var ntpBlockParts []string
-
-	for _, s := range servers {
-		ntpBlockParts = append(ntpBlockParts, fmt.Sprintf("NTP=%s", s.String()))
+// FindNodeCondition returns the condition of the given type from the node. The second return value indicates if the condition was found.
+func FindNodeCondition(node corev1.Node, conditionType corev1.NodeConditionType) (corev1.NodeCondition, bool) {
+	for _, condition := range node.Status.Conditions {
+		if condition.Type == corev1.NodeReady {
+			return condition, true
+		}
 	}
+	return corev1.NodeCondition{}, false
+}
 
-	ntpBlock := strings.Join(ntpBlockParts, "\n")
-
-	return ntpBlock
+// FindNodeCondition returns the condition of the given type from the pod. The second return value indicates if the condition was found.
+func FindPodCondition(pod corev1.Pod, conditionType corev1.PodConditionType) (corev1.PodCondition, bool) {
+	for _, condition := range pod.Status.Conditions {
```

Comment:
`conditionType` is not being used in the function. I'm assuming you want to check for it here?